src/js/components/Select/doc.js

```diff
@@ -26,6 +26,23 @@ export const doc = Select => {
     closeOnChange: PropTypes.bool
       .description('Wether to close the drop when a selection is made.')
       .defaultValue(true),
+    clear: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({
+        position: PropTypes.oneOf(['top', 'bottom']).description(
+            `Add a clear value to the top or at the bottom of the
+            container. By default no unselect option is present.`,
+        ),
+        label: PropTypes.string
+          .description('Label for the clear selection item')
+          .defaultValue('Clear selection'),
+        renderValue: PropTypes.func.description(
+          `Render function to customize clear value item.
+          It receives as a parameter the callback to call
+          when clicking on the item`,
+        ),
+      }),
+    ]).description(`Whether to provide button to clear selection.`),
```

Comment:
we should add that `clear` has a defaultValue of `false`.

Connected to this comment:
![image](https://user-images.githubusercontent.com/6320236/75051265-c7a63500-548a-11ea-9df7-a92151143275.png)